src/js/game/LevelMVC/LevelModel.js

```diff
@@ -51,7 +51,8 @@ module.exports = class LevelModel {
       if (this.actionPlane[i].blockType.substring(0,12) === "redstoneWire") {
         let y = Math.floor(i / this.planeHeight);
         let x = i - (y * this.planeHeight);
-        this.determineRedstoneSprite(x, y, this.actionPlane[i]);
+        let position = [x,y];
+        this.actionPlane.determineRedstoneSprite(position, i);
```

Comment:
I think we want to trigger this whenever `LevelPlane.prototype.setBlockAt` is called.